common-types/index.d.ts

```diff
@@ -87,7 +87,7 @@ interface TeamEventAttendance {
   hoursAttended?: number;
   image: string;
   readonly eventUuid: string;
-  readonly pending: boolean;
+  readonly pending?: boolean;
```

Comment:
Same here as well, feel free to remove this field entirely, since we've fully migrated over to the `status` field.